### ISSUE TITLE
Syntax error with python 3.7

### ISSUE THREAD

**Issue by lovetox** [enthought/pywin32-ctypes#81]
After installing python 3.7 i get this when using keyring
```
Traceback (most recent call last):
  File "C:\msys64\mingw32\lib\python3.7\site-packages\keyring\backend.py", line 148, in _load_plugins
    init_func = ep.load()
  File "C:\msys64\mingw32\lib\python3.7\site-packages\entrypoints.py", line 77, in load
    mod = import_module(self.module_name)
  File "C:\msys64\mingw32\lib\python3.7\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\msys64\mingw32\lib\python3.7\site-packages\keyring\backends\Windows.py", line 12, in <module>
    from win32ctypes.pywin32 import pywintypes
  File "C:\msys64\mingw32\lib\python3.7\site-packages\win32ctypes\pywin32\__init__.py", line 21, in <module>
    from win32ctypes.pywin32 import win32api
  File "C:\msys64\mingw32\lib\python3.7\site-packages\win32ctypes\pywin32\win32api.py", line 23, in <module>
    from win32ctypes.core import _common, _kernel32, _backend
  File "C:\msys64\mingw32\lib\python3.7\site-packages\win32ctypes\core\__init__.py", line 39, in <module>
    from .cffi import _advapi32, _common, _kernel32
  File "C:\msys64\mingw32\lib\python3.7\site-packages\win32ctypes\core\cffi\_advapi32.py", line 198

    ^
SyntaxError: invalid syntax
```
And indeed on line 198 is a newline to much, after fixing this everything worked.

I wanted to make a Pull request, but this file is not in the repo, probably build somehow?

**Comment by bh1428** [comment]
Having the same issue (Python 3.7.0 on Windows). I did some further analysis and it seems to be caused by 2to3 (which runs during installation of pywin32-ctypes). The extra newlines are generated by 2to3 and that is a bug, see [Python 3.7 changelog](https://docs.python.org/3/whatsnew/changelog.html#changelog) or more specifically [bpo-34108: Remove extraneous CR in 2to3 refactor](https://bugs.python.org/issue34108). I suppose that will be fixed in Python 3.7.1...

Meanwhile, as a workaround, you can remove the extra newlines.

**Comment by lovetox** [comment]
How is this a workaround? I cant tell users to go to the source files and remove some line

**Comment by bh1428** [comment]
FYI: I'm not a maintainer of pywin32-ctypes, keyring and/or Pythons 2to3. Furthermore: I have no time or energy to investigate it further or provide a PR, but thought it might be useful to share what I know so far.

Anyway, my personal workaround is to run Python 3.6 and 3.7 in parallel (within their own pipenv environments), so I just run keyring in Python 3.6.

**Comment by dumbmud** [comment]
This isn't an issue with the master branch, tested it out and it seemed to work fine.

**Comment by jaraco** [comment]
The [2to3 issue is here](https://bugs.python.org/issue34108).

**Comment by jaraco** [comment]
It might be worth considering dropping reliance on 2to3.

**Comment by itziakos** [comment]
Sorry for the delay, for some reason, I missed the notification about the issue. Will look into it amd make a release asap

**Comment by itziakos** [comment]
There are a number of issues with the test builds and probable issues with python 3.7 (#82). I Will need to investigate (probably over the weekend).

**Comment by itziakos** [comment]
PR #82 was merged (it looks that there some problems with the python 3.7 build of pywin32). I will prepare a release PR and have a pypi release by tomorrow.

**Comment by itziakos** [comment]
A new release 0.2.0 is now available and tests pass for python 3.7 (both 32 and 64 bit)

**Comment by bjones1** [comment]
Thanks for doing this! My project (PyInstaller) depends on this, and it helps me get Python 3.7 working. I appreciate it.
